lib/manager/bitbucket-pipelines/extract.ts

```diff
@@ -0,0 +1,64 @@
+import { logger } from '../../logger';
+import { getDep } from '../dockerfile/extract';
+import type { PackageDependency, PackageFile } from '../types';
+
+const pipeRegex = /^\s* - pipe:\s*'?"?([^\s'"]+)'?"?\s*$/;
+const dockerImageRegex = /^\s*-?\s?image:\s*'?"?([^\s'"]+)'?"?\s*$/;
+
+export function extractPackageFile(content: string): PackageFile | null {
+  const deps: PackageDependency[] = [];
+
+  try {
+    const lines = content.split('\n');
+    for (let lineNumber = 0; lineNumber < lines.length; lineNumber += 1) {
+      const line = lines[lineNumber];
```

Comment:
```suggestion
    for (const line of lines) {
```

Simplify, we don't need `lineNumber`